### PR TITLE
added file

### PR DESCRIPTION
**Pull request by mccue**
This pull request adds a folder and file for san jose @mccue :tropical_fish: 
This pull request addresses #66

### CODE REVIEW

File: mccue/mccue-sanjose.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# San Jose
+
+### Restaurants
+
+### Things to do
```
Comment (MGineer) [inline mccue/mccue-sanjose.md:5]: @mccue I like sports what would you recommend in San Jose?

**Comment by prachi21** [comment]
:+1:

File: mccue/mccue-sanjose.md (hunk 1 above)
Comment (mccue) [inline mccue/mccue-sanjose.md:5]: The Earthquakes is the local football (soccer) team. If you like BB, you may have watched the Warriors :unamused:

**Comment by mccue** [comment]
@MGineer @prachi21 Thanks for your comments! Merci!

**Pull request merged**